files/setup-04-kubernetes.sh

```diff
@@ -54,4 +54,10 @@ while [[ $(systemctl is-active kubelet.service) == "inactive" ]]
 do
     echo -e "\e[92mk8s service is still inactive, sleeping for 10secs" > /dev/console
     sleep 10
-done
\ No newline at end of file
+done
+
+echo -e "\e[92mDeploying Local Storage Provisioner ..." > /dev/console
+mkdir -p ${LOCAL_STOARGE_VOLUME_PATH}/local-path-provisioner
+chmod 777 ${LOCAL_STOARGE_VOLUME_PATH}/local-path-provisioner
+kubectl --kubeconfig /root/.kube/config apply -f /root/download/local-path-storage.yaml
```

Comment:
nit: in future we might replace these with an environment variable (`KUBECONFIG`)

Reply:
Do you have an example of what you'd like to see?

Reply:
If you set `KUBECONFIG` (https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable), your commands can be used without `--kubeconfig /root/.kube/config`